test/testServer.js

```diff
@@ -0,0 +1,23 @@
+let server = null;
+
+module.exports = {
+  onPrepare: function() {
+    return new Promise(function(resolve) {
+      const webpack = require('webpack');
+      const middleware = require('webpack-dev-middleware');
+      const express = require('express');
+      const devRoutes = require('../server/dev');
+      const app = express();
+      const config = require('../webpack.config');
+      const wpm = middleware(webpack(config), { logLevel: 'silent' });
+      app.use(wpm);
+      devRoutes(app, { middleware: wpm });
+      wpm.waitUntilValid(() => {
+        server = app.listen(8081, resolve);
+      });
+    });
+  },
+  onComplete: function() {
+    server.close();
+  }
+};
```

Comment:
This creates a web server that has all of the assets and images loading? That was one problem I ran into. I wonder if I could use this for my tests as I had to add a script to ignore the ```.dockerignore``` so I could run the  app with all the assets.